BeforeBuild.cmd

```diff
@@ -8,8 +8,23 @@ SET NuGetPackageDir=src\packages
 
 md bld\bin\nuget
 
+::Restore nuget packages
+%~dp0.nuget\NuGet.exe restore src\Sarif.Viewer.VisualStudio\Sarif.Viewer.VisualStudio.csproj -ConfigFile "%NuGetConfigFile%" -OutputDirectory "%NuGetPackageDir%"
+
+if "%ERRORLEVEL%" NEQ "0" (
+echo NuGet restore failed.
+goto ExitFailed
+)
+
+%~dp0.nuget\NuGet.exe restore src\Sarif.Viewer.VisualStudio.UnitTests\Sarif.Viewer.VisualStudio.UnitTests.csproj -ConfigFile "%NuGetConfigFile%" -OutputDirectory "%NuGetPackageDir%"
+
+if "%ERRORLEVEL%" NEQ "0" (
+echo NuGet restore failed.
```

Comment:
Might be worth saying `NuGet restore failed for project Sarif.Viewer.VisualStudioUnitTests.`.

Reply:
Done